networkit/distance.pyx

```diff
@@ -1841,3 +1841,70 @@ cdef class PrunedLandmarkLabeling(Algorithm):
 			The shortest-path distances from the source node to the target node.
 		"""
 		return (<_PrunedLandmarkLabeling*>(self._this)).query(u, v)
+
+cdef extern from "<networkit/distance/DynPrunedLandmarkLabeling.hpp>":
+
+	cdef cppclass _DynPrunedLandmarkLabeling "NetworKit::DynPrunedLandmarkLabeling"(_Algorithm):
+		_DynPrunedLandmarkLabeling(_Graph G) except +
+		count query(node u, node v) except +
+		void update(_GraphEvent ev) except +
+
+cdef class DynPrunedLandmarkLabeling(Algorithm):
```

Comment:
For the Python implementation, we should wait until #1067 is merged. The PR slightly changes how Cython code for subclasses of `DynAlgorithm` should be written.

Reply:
Yes, it's great that now we can inherit from multiple classes in Cython. I rebased and updated the class accordingly.